### ISSUE TITLE
Improved "Section" UI in PR environment and get feedback

### ISSUE THREAD

**Issue by rachel-kauff** [department-of-veterans-affairs/va.gov-cms#5088]
# Background 

Comes from preparatory work for user admin service discovery. See #4861.

To streamline our interface language and avoid confusion, we should stop using the word "Governance" and "Owner" and use "Section" instead.

This relatively minor UI change (from an engineering perspective) will affect every CMS user and will require careful design review and change management. 

**ACs**
- [x] [a coded prototype](https://pr5151-h6rmtqwhvbrhne87fl20zbdxq1g1whvw.ci.cms.va.gov/) (PR environment with draft PR)  @kevwalsh 
  - [x] Labels and help text updated to match prototype for **every form** where the governance widget appears.
  - [x] Change column label from Owner to Section in every content table where it appears. 
  - [ ] Stretch: Rename "Workbench Access" tab on user profile to "Sections".
- [x] Draft release note written
- [ ] Review 
  - [x] Design @rachel-kauff 
  - [x] Help desk - scheduled
  - [x] Training @xiongjaneg - scheduled
  - [ ] Some users 
    - [ ] a user with access to one section - TBD
    - [ ] a VISN manager with access to more than one VAMC (other than LS) - inquiry sent
    - [x] a content admin (OCTO-DE) - scheduled
    - [x] @jenniferlee-dsva - scheduled

Stretch 
- [ ] UX writing for Media "reusable" field label
- [ ] UX writing for special treatment for any reusable content (alert blocks, COPE FAQs)

## Artifacts
[WIP design specs](https://www.figma.com/file/CM9J6YLLF9W0GMdYsbiVRH/Sections-menu?node-id=0%3A1)

**Comment by rachel-kauff** [comment]
Prototype:

![Screen Shot 2021-04-22 at 3.29.29 PM.png](https://images.zenhubusercontent.com/5e399cf5446c5b4ca0e31e42/58767f2d-ab93-490a-af5b-14429cd88538)

**Comment by rachel-kauff** [comment]
From Stan: Editors with VISN level access: 
- Jim Hoehn	james.hoehn2@va.gov	VISN 19
- Matthew Gowan	Matthew.Gowan@va.gov	VISN 16
- Stephen Wilkins	Stephen.Wilkins2@va.gov VISN
- Tara Ricks	Tara.ricks@va.gov	VISN 6

**Comment by kevwalsh** [comment]
@rachel-kauff This PR environment is ready for review. It's mergeable code, although we won't update some automated tests until this gets the green light.

**Comment by rachel-kauff** [comment]
@kevwalsh Draft of CMS announcement content:

@ here 🔍 **Section = where you can find and edit content**

Recently we made a few small changes so that the CMS uses the term "Sections" in a way that's consistent and clear. You may or may not notice them, but here's a reminder of what you need to know about this important CMS concept:

* All CMS content is belongs to one or more Sections
* Each CMS editor is assigned to a section based on what kind of content they work on. 
* When creating a brand new piece of content, you may need to select which section it belongs to if you work on more than one section. 
* You can see what Section a piece of content belongs to when searching for content. 


That's all for now. Still seeking some serious Sections schooling? Let us know in the :thread:.”

**Comment by jenniferlee-dsva** [comment]
@kevwalsh @rachel-kauff - Can we add some info here as well, wrt how sections and permissions work (currently)?  This is the concept that most content admins still don't quite understand. Or at least explicitly call out that "section" doesn't equal "stakeholder" or "SME business owner"? 

I will give this a think and send over some suggestions. I have a massive deck on this which I used to already explain sections idea to sitewide team but they were not all in attendance. Anyway, may be able to pull something from there. 

This is a great beginning though!

**Comment by jenniferlee-dsva** [comment]
Something like this - feel free to refine/format however. 

1/
Sections don't always correspond to the stakeholder or business SME, and should not be used interchangeably. 

Example: 
Root level pages are in the Veterans Affairs OPIA owned "section." But root level pages are actually owned by DEPO ( `Change your address `page,` Direct deposit `page, `Records`, etc.). 

2/ 
About how sections and user roles combine to determine what activities a user can perform in the CMS: 

- A. You can have more than 1 user role. 
- B. You can be assigned to more than 1 section. 
- You cannot be assigned a specific role for each section. I.e., whatever user role (or roles) you have in A, you will have those permissions/abilities for ALL the sections you have access to.
- In other words, a VBA user cannot have a publisher role for the Outreach hub, but only an editor for the benefit hubs. If they have access to both sections, they will have the same roles for both.

**Comment by jenniferlee-dsva** [comment]
Can we also add that if you have any doubt as to which section a page/node should be assigned, to please reach out? 

I've seen content ppl make some understandably erroneous section assigmments - like putting  health related RS articles into the "VHA" section "owner," rather than under DEPO.  It's easy to conflate the business SME as the section when it's not quite the same thing.  

(Future state - we should not have content admins assigning sections per each node anyway. We should -- ideally  -- have a content type be assigned at the beginning to a section, and the section field is just autopopulated and not editable by individual users.)

**Comment by rachel-kauff** [comment]
@jenniferlee-dsva Thank you for this additional context. I'll revise the announcement to include a link to a Help guide where people can learn more about these nuances.

**Comment by rachel-kauff** [comment]
@kevwalsh could we add "lightweight improvements to [Roles and permissions help guide](https://prod.cms.va.gov/help/cms-basics/roles-and-permissions)" for next sprint to go along with the announcement?

**Comment by kevwalsh** [comment]
Closing this, design is [here](https://www.figma.com/file/HpxCd7YNEVG2Np1wddg6LW/Add-user---WIP?node-id=114%3A0)

> @kevwalsh could we add "lightweight improvements to Roles and permissions help guide" for next sprint to go along with the announcement?

incorporated in #5251
